### PR TITLE
Mtrebron newsimages

### PR DESCRIPTION
**Pull request by mtrebron**
Add a dependency on plone.app.imagecropping to enable layout updates in ploneorg.theme

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -52,7 +52,8 @@
         'simplejson',
         'twitter',
         'plone.app.mosaic',
-        'pyquery'
+        'pyquery',
+        'plone.app.imagecropping'
```
Comment (jensens) [inline setup.py:56]:
```suggestion
        'plone.app.imagecropping',
```
in future, avoid two line changes in such a case. (the whole code could get a black/isort overhaul..., but that's OT here.)

File: ploneorg/core/browser/foundationsponsor.pt [hunk 2]
```diff
@@ -9,49 +9,192 @@
...
+                  (<span tal:content="context/country">Country</span>)
+                </tal:country>
+              </h4>
+            </div>
+        </tal:hide>
+      </div>
     </div>
 
-    <div class="logo">
+    <div class="logo" tal:condition="nothing">
```
Comment (jensens) [inline ploneorg/core/browser/foundationsponsor.pt:197]: Do not be afraid to remove code - we have Git. Or do you need it for debugging? then enabled it i.e. for "Manage Portal" permission.

**Comment by jensens** [comment]
plone.app.imagecropping can be configured properly via registry.xml, see 

https://github.com/collective/plone.app.imagecropping/blob/82d5787f7d003fed1ae9a3464671b142b43e88a3/src/plone/app/imagecropping/browser/settings.py#L28-L44

**Comment by fulv** [comment]
> plone.app.imagecropping can be configured properly via registry.xml, see
> 
> https://github.com/collective/plone.app.imagecropping/blob/82d5787f7d003fed1ae9a3464671b142b43e88a3/src/plone/app/imagecropping/browser/settings.py#L28-L44

We will configure the crop sizes manually within @@imagecropping-settings for now, until we have found a set of sizes we are happy with.  Later, we will configure them via registry.xml.

**Pull request merged**